### PR TITLE
Added some of the SteamClient types

### PR DESCRIPTION
**Pull request by Tormak9970**

### CODE REVIEW

**Comment by AAGaming00** [comment]
Please run prettier.

**Review by AAGaming00 — CHANGES_REQUESTED** [review]
Please run prettier.

**Review by AAGaming00 — DISMISSED** [review]
Please use declare global to declare this as global in index.ts. not sure if it works for those importing but hopefully it does.

**Comment by FrogTheFrog** [comment]
@Tormak9970 this comment is a little bit incorrect `unAppID: number; // seems to be 0 for shortcuts :/`. Currently there is a bug in gamescope/steamclient where it does not properly assign the app id to the process (it tries to, but fails). As a result the app id remains 0.

I think it should be changed to `// currently it is always 0 for shortcuts, due to a bug in gamescope/steamclient`

**Comment by AAGaming00** [comment]
It's a bug in steam, not gamescope 
Please link [this](https://github.com/ValveSoftware/steam-for-linux/issues/8474) in the comment

Also please convert any important comments to JSDoc (/**) format so they show up in the docs on the wiki.

**Comment by Tormak9970** [comment]
Changed it to a JSDoc prop comment and declared the global

File: src/deck-components/index.ts [hunk 1]
```diff
@@ -21,3 +21,10 @@ export * from './Tabs';
 export * from './TextField';
 export * from './Toggle';
 export * from './ToggleField';
+export * from './SteamClient';
+
+import {SteamClient} from './SteamClient'
+
+declare global {
+    var SteamClient: SteamClient;
+}
```
Comment (JohnnyCrazy) [inline src/deck-components/index.ts:30]: missing newline here

Comment (JohnnyCrazy) [inline src/deck-components/index.ts:30]: Does does TypeScript handle it correctly if my current file does not import anything from `decky-frontend-lib`? Like, will the globals still be recognized?

I will try it out later, but maybe one of you knows already...

Comment (Tormak9970) [reply src/deck-components/index.ts:30]: I don't know, it was suggested by @AAGaming00

Comment (AAGaming00) [reply src/deck-components/index.ts:30]: Likely not, but I don't see why that's an issue.

Comment (JohnnyCrazy) [reply src/deck-components/index.ts:30]: Yea I guess you always have the import-chain from the file where `definePlugin` was used, so besides the newline, LGTM

**Comment by JohnnyCrazy** [comment]
I think besides the missing newline, we would be good to go @AAGaming00 ?

File: src/deck-components/index.ts (hunk 1 above)
Comment (TrainDoctor) [inline src/deck-components/index.ts:30]:
```suggestion
}

```

**Comment by TrainDoctor** [comment]
@AAGaming00 Please review.

**Review by AAGaming00 — APPROVED** [review]

**Pull request merged**